### PR TITLE
RET-5243

### PR DESCRIPTION
**Pull request by cindychmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RET-5243

### Change description ###

When I'm in the 'Reply to a referral' event, I want to be able to open the DCF

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by lvo700 — APPROVED** [review]

**Review by hsjhita — DISMISSED** [review]

**Review by hsjhita — APPROVED** [review]
Configs need to be merged into master and uploaded to prod before this can be merged in so just let me know when I can upload them

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DigitalCaseFileService.java
Comment (fergusoshea) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DigitalCaseFileService.java:129]: Wouldn't "TRIBUNAL_CASE_FILE.equals(d.getValue().getMiscDocuments())" do the same thing without adding the extra import?

**Review by fergusoshea — APPROVED** [review]

**Pull request merged**
